docs/_drafts/go-strategy-pattern-a-response.md

```diff
@@ -0,0 +1,172 @@
+---
+title: The Strategy Pattern In Go -- A Response
+tags: golang patterns
+---
+
+This is a response to the recent article [Strategy pattern in Go](https://rednafi.com/go/strategy_pattern/) by Redowan ("rednafi").
+I like the author's easy-to-understand example of string formatting methods and their contrast of Ruby code with Go.
+With that said, their solution feels verbose and overcomplicated to me, so I wanted to address that and propose some alternatives.
+
+## Background -- the Strategy Pattern
```

Comment:
:wrench: 
```suggestion
## Background --- the Strategy Pattern
```

Kramdown should detect `---` as an em-dash but not `--`

Reply:
Fixed in https://github.com/agrski/agrski.github.io/pull/45/commits/78770ff1b17f54bd7f15cf234f37c28d217b8806